scripts/ci/libraries/_verify_image.sh

```diff
@@ -340,7 +340,7 @@ function verify_image::verify_prod_image_as_root() {
 function verify_image::verify_production_image_has_dist_folder() {
     start_end::group_start "Verify prod image has dist folder (compiled www assets): ${DOCKER_IMAGE}"
     # shellcheck disable=SC2016
-    verify_image::check_command "Dist folder" '[ -f $(python -m site --user-site)/airflow/www/static/dist/manifest.json ] || exit 1'
+    verify_image::check_command "Dist folder" '[ -f /.venv/lib/python$(python -c "import sys; print(str(sys.version_info[0])+\".\"+str(sys.version_info[1]))")/site-packages/airflow/www/static/dist/manifest.json ] || exit 1'
```

Comment:
Probably easier to use `importlib.resources` instead of manually constructing the path.